features/networking/multus.feature

```diff
@@ -470,14 +470,14 @@ Feature: Multus-CNI related scenarios
     # Create the net-attach-def via cluster admin
     Given I switch to cluster admin pseudo user
     And I use the "default" project
-    When I run the :create admin command with:
+    When I run the :create client command with:
 	    | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/macvlan-bridge.yaml |
     Then the step should succeed
     Given I register clean-up steps:
     """
     Given I switch to cluster admin pseudo user
     And I use the "default" project
-    When I run the :delete admin command with:
+    When I run the :delete client command with:
```

Comment:
@anuragthehatter, indentation should be 2, not 4.

Reply:
Thanks @pruan-rht i knew that but somehow missed it. Also i need to work on my auto indent vim feature

Reply:
https://github.com/openshift/verification-tests/blob/master/doc/vim.adoc